pandas-stubs/core/frame.pyi

```diff
@@ -656,7 +656,7 @@ class DataFrame(NDFrame, OpsMixin, _GetItemHack):
         show_counts: bool | None = ...,
     ) -> None: ...
     def memory_usage(self, index: _bool = ..., deep: _bool = ...) -> Series: ...
-    def transpose(self, *args, copy: _bool = ...) -> Self: ...
+    def transpose(self, *args: Any, copy: _bool = ...) -> Self: ...
```

Comment:
`args` are ignored, ok to use `Any` here?

Reply:
yes